### PR TITLE
doc : missing link in "quick_guide.rst"

### PR DESCRIPTION
**Pull request by Oxygen31**
Fixes: #12009

Signed-off-by: Germain CHIPAUX <germain.chipaux@gmail.com> 
Signed-off-by: Thomas LAUMONDAIS <thomas.laumondais@gmail.com>
Signed-off-by: Pierre CHAUMONT <pierre.chaumont31@gmail.com>
Signed-off-by: Benjamin TROQUEREAU <benjamin.troquereau@gmail.com>
Signed-off-by: Benjamin SESIA <benjamin.sesia@gmail.com>

### CODE REVIEW

File: doc/dev/quick_guide.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ the result behaves as expected.
 
 Running a development deployment
 --------------------------------
-Ceph contains a script called ``vstart.sh`` which allows developers to quickly test their code using
+Ceph contains a script called ``vstart.sh`` (consult document on `deploying a development cluster<https://ceph.com/docs/master/dev/dev_cluster_deployement/>`_ for more details) which allows developers to quickly test their code using
```
Comment (dachary) [inline doc/dev/quick_guide.rst]: s/consult document on/see also/

Comment (dachary) [inline doc/dev/quick_guide.rst]: s/for more details//

**Comment by dachary** [comment]
Could you please make one Signed-off-by: line per mail ?

**Comment by Oxygen31** [comment]
@dachary We didn't understand what you meant with "s/consult document on/see also/" and "s/for more details//"... What do you want ? 
Thanks.

**Comment by dachary** [comment]
@Oxygen31 the **s/foo/bar/** is a short way to say **replace foo with bar**. It's like using a vim command :-)

**Pull request closed without merging**